types/mongodb/index.d.ts

```diff
@@ -36,7 +36,7 @@ export class MongoClient {
 }
 
 export interface MongoCallback<T> {
-    (error: MongoError, result: T): void;
+    (error: MongoError, result: T | null): void;
```

Comment:
`MongoCallback` is used in many other signatures. `result` should remain of type `T`. Change the signature of the `findOne` function to `MongoCallback<T|null>` instead.

Reply:
done :+1: